protocol/src/main/protobuf/Transactor.proto

```diff
@@ -0,0 +1,27 @@
+syntax = "proto3";
+package protocol;
+option java_package = "io.mediachain.protocol.transactor";
+
+message MultihashReference {
+    string reference = 1;
+}
+
+message InsertRequest {
+    bytes canonicalCbor = 1;
+}
+
+message UpdateRequest {
+    bytes chainCellCbor = 1;
+}
+
+service TransactorService {
+    rpc FetchObjectChainHead(MultihashReference) returns
```

Comment:
The prime chain in user context is the mediachain for an entity or an artefact.
The user of the transactor does not care about the blockchain in this interface so there shouldn't be
an confusion.